src/models/local-user.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+module.exports = class LocalUser {
```

Comment:
I thought that having a model for TL users was a good idea, but I'm not really sure.
It's useful for things like `isAdmin()` helpers, otherwise they have to live in `client.js` otherwise, already pretty bloated.
If this is a good pattern we want to keep, I can see more and more things using `LocalUser` over time, which will be a subset of `client.js`.
Feedback welcome.

Reply:
@xPaw, what's your take on this?